test/verify/machineslib.py

```diff
@@ -1408,11 +1411,29 @@ def installWithErrorTest(dialog):
                                          location=config.VALID_URL, storage_size=1,
                                          delete=False))
 
-        checkDialogFormValidationTest(TestMachines.VmDialog(self, "existing-name", storage_size=1,
-                                                            env_is_empty=False), {"Name": "already exists"})
-
         self.machine.execute("virsh undefine existing-name")
 
+        # name already used from a VM that is currently being created
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1780451
+        if self.machine.image in ['debian-stable', 'debian-testing', 'ubuntu-stable', 'ubuntu-1804', 'fedora-30', 'fedora-testing', "centos-8-stream"]:
```

Comment:
This OS list is unclear - the bz is against cockpit, and that changes the logic within cockpit. Why would it be specific to F31, and not work on F30 or Debian? Does this depend on a fix in a recent libvirt? The comment should be clarified here, as most probably the list needs to be updated on image refreshes in the future, and we need to know what's going on.

Reply:
So this BZ is about a VMs trying to be created with the same name as a VM that currently being created. In order to have this long running creation process (basically this last while virt-install tries to download the initrd and vmlinuz files from the installation source) I chose to use the installation option from the dialog 'Download an OS' which will try to use the official fedora download mirror. Without network this is of course in accessible to that will take ages. However this option is not available with old virt-install versions and that's why I have this check here. If it's still unclear I will add a comment.

Reply:
Ah, thanks. Can you please add something like `# needs virt-install version ≥ 1.2.34 for feature blah`?